src/app/core/components/top-right-menu/top-right-menu.component.ts

```diff
@@ -10,13 +13,22 @@ export class TopRightMenuComponent {
 
   readonly menuItems = [
     { label: 'Profile', icon: 'pi pi-user', routerLink: [ 'groups', 'me' ] },
+    ...(!appConfig.production
+      ? [{ label: 'Invalidate token', icon: 'pi pi-refresh', command: (): void => this.invalidateToken() }]
+      : []
+    ),
```

Comment:
you can probably leave this in the code, just move this is a function not to pollute the readability of this list too much.

Reply:
Fixed